server/views/pages/assessEmail.njk

```diff
@@ -58,6 +60,9 @@
                                     classes: 'govuk-fieldset__legend--m'
                                 }
                             },
+                            hint: {
+                                text: 'For example, 23 5 2021 at 15:46'
```

Comment:
Tiny suggestion:  is the time of day entered like "15:46"?  I thought I'd seen more like "15 46" - as separate fields?
This hint might also be a good place to mention e.g. "(use 24 hour clock for hours)" or similar.

Reply:
Just copying what's in the prototype; I see your point though
Yep I'd mentioned about 24 hour clock before but the decision was to use the hint text above